coralproject/talk
# Update scraper to also pull section from `meta[itemprop="articleSection"]`

## What does this PR do?

Updates the scraper to pull the schema.org defined `articleSection` meta tag.

This code is already in a 6.18.2 hotfix release and is currently being deployed to SaaS customers that aren't upgraded to 7.x versions yet. We want this fix in all of Coral, so this brings it into develop so it can carry on into Coral 7+.

6.18.2 version of this fix: https://github.com/coralproject/talk/pull/3978

**Why didn't we merge the 6.18.2 branch directly into develop?** 

Because we hotfixed this on top of a previous release (6.18.1) that has already been surpassed by our versioning (Coral is on 7+) right now. So we are currently maintaining a `main-6x` branch for older clients until they are eventually migrated to Coral 7+. If we merged in 6.18.2, it would collide on our `package.json` on the version number. It makes no sense to merge a branch from behind us in time into our main and develop branches. So this has to come in as a separate PR.

This commit is still cherry picked from that 6.18.2 release and its commit will match the identifier of that commit. So there is consistency when diffing the two main branches, should you ever need to do it.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

See testing notes for the 6.18.2 version of this fix: https://github.com/coralproject/talk/pull/3978
 
## How do we deploy this PR?

No special considerations.
